src/nccl_ofi_net.c

```diff
@@ -948,10 +948,22 @@ static ncclResult_t ofi_connect(int dev, void *handle, void **sendComm)
 		goto error;
 	}
 
+	req = allocate_nccl_ofi_request(sComm->nccl_ofi_reqs_fl);
```

Comment:
Great that you changed this! I was about to point this out in case we were unable to send request in single go.